.github/workflows/pr_agent.yml

```
@@ -0,0 +1,24 @@
+name: AI PR Review
+
+on:
+  pull_request:
+
+jobs:
+  review:
+    runs-on: ubuntu-latest 
+    steps:
```

[ReviewCategory.STYLE] The `actions/checkout@v3` action uses Node.js 16, which is deprecated and will be removed from GitHub-hosted runners. It is recommended to update to `v4` which uses Node.js 20. See https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/ for more details.

```suggestion
      - uses: actions/checkout@v4
```